### PR TITLE
Export `categorize_risk` function

### LINKED ISSUE #792: Export function `categorize_risk` for risk classification of transition risk scores
Relates to 
- https://github.com/2DegreesInvesting/tiltIndicatorAfter/issues/241

The function `categorize_risk` should be exported for risk classification of transition risk scores in tiltIndicatorAfter/tiltTranistionRisk.

### PR DESCRIPTION
**Pull request by kalashsinghal**
closes #792 

Dear @maurolepore, I need this function to complete work on risk classification of transition risk scores (related to [Issue](https://github.com/2DegreesInvesting/tiltIndicatorAfter/issues/241)). Please review!

----

TODO

- [ ] [Link related issue/PR]([url](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)). 
- [ ] Describe the goal of the PR. Avoid details that are clear in the diff.
- [ ] Mark the PR as draft.
- [ ] [Include a unit test](https://code-review.tidyverse.org/reviewer/aspects.html#sec-tests).
- [ ] Review your own PR in "Files changed".
- [ ] Ensure the PR branch is updated.
- [ ] Ensure the checks pass.
- [ ] Change the status from draft to ready.
- [ ] Polish the PR description to reflect what it ended up doing.
- [ ] Polish the PR title as you'd like others to read it from the git log.
- [ ] Assign a reviewer.
- [ ] Document user-facing changes in the changelog.

### CODE REVIEW

File: R/utils.R [hunk 1]
```diff
@@ -14,6 +14,13 @@ risk_category_levels <- function() {
   c("high", "medium", "low")
 }
 
+#' Categorize variable into risk categories using low and high thresholds.
```
Comment (maurolepore) [inline R/utils.R:17]:
```suggestion
#' Categorize a variable into risk categories using low and high thresholds
```

File: R/utils.R [hunk 2]
```diff
@@ -14,6 +14,13 @@ risk_category_levels <- function() {
...
+#'
+#' @param x variable.
+#' @param low_threshold Dataframe column.
+#' @param high_threshold Dataframe column.
```
Comment (maurolepore) [inline R/utils.R:21]:
```suggestion
```

Comment (maurolepore) [reply R/utils.R:21]: This is a thin wrapper around `dplyr::case_when()`, and very similar to `ifelse()` -- so the thresholds can but don't have to be columns.

``` r
tiltIndicator:::categorize_risk(c(1:30), 10, 20)
#>  [1] "low"    "low"    "low"    "low"    "low"    "low"    "low"    "low"   
#>  [9] "low"    "low"    "medium" "medium" "medium" "medium" "medium" "medium"
#> [17] "medium" "medium" "medium" "medium" "high"   "high"   "high"   "high"  
#> [25] "high"   "high"   "high"   "high"   "high"   "high"
```


Also I suggest removing the documentation. As a developer-oriented function we expect the user to be able to read source code. This makes it easier to maintain.

Else I would document it in full -- adding for example documentation for `...` and also examples.

**Review by maurolepore — APPROVED** [review]
LGTM but please see mu comment and suggested changes.

File: R/utils.R (hunk 2 above)
Comment (kalashsinghal) [reply R/utils.R]: Makes sense! I have removed the suggested documentation. Thanks for the review!

**Pull request merged**